src/components/titles/index.js

```diff
@@ -97,8 +99,10 @@ function draw(gd, titleClass, options) {
         if(!editable) txt = '';
     }
 
-    if(fullLayout.meta) {
-        txt = Lib.templateString(txt, {meta: fullLayout.meta});
+    if(options._meta) {
```

Comment:
Wondering if we need to bypass `[ ]` or `{ }` here?

Reply:
No need. `Lib.termplateString` handles those cases correctly.